source/MapDetailPanel.cpp

```diff
@@ -472,73 +530,29 @@ void MapDetailPanel::DrawInfo()
 		PointerShader::Draw(uiPoint + Point(-90., 20.), Point(1., 0.),
```

Comment:
I would like to make the government sprite draw after so planets can be partially hidden by it
It would however completely break the uiPoint thing (I mean all elements will be made constant in position except for the ones in the mapPlanetCards), but I have to make the tradeY constant anyway(as in defined in the interface and not dynamic) so it should not be a problem?

Reply:
I made it be drawn after, and I linked the trade sprite position to the end of the planets